test/standalone/Bucket/soil_bucket_tests.jl

```diff
@@ -98,6 +98,26 @@ for bucket_domain in bucket_domains
         )
         # Initial conditions with no moisture
         Y, p, coords = initialize(model)
+        # test if the correct dss buffers were added to aux
+        if typeof(model.domain) <: Union{
```

Comment:
could you add a comment explaining why we expect buffers for these 3d domains but not LSMSingleColumnDomain please?